src/main/java/org/commcare/suite/model/MenuLoader.java

```diff
@@ -42,6 +42,16 @@ public MenuLoader(CommCarePlatform platform, SessionWrapperInterface sessionWrap
         this.getMenuDisplayables(platform, sessionWrapper, menuId, hideTrainingRoot);
     }
 
+    public MenuLoader(CommCarePlatform platform, SessionWrapperInterface sessionWrapper,
```

Comment:
Can the pre-existing constructor use the convention of calling this one and passing a value for `includeBadges`? It's not a lot of duplicate code, but it's going to be easy for a future dev to update one of the contructors instead of both.